### PR TITLE
Sync various section

### PR DESCRIPTION
**Pull request by victor-prdh**
Hello, 
i synced some sections

Thanks for the proofreading :)

### CODE REVIEW

**Comment by victor-prdh** [comment]
Just a little question, is that better for you than i open a PR and keep adding content during it's not merged or stop pushing content after opening the PR and keep the next translations for the next PR ?

**Comment by adoy** [comment]
I think it's easier if once someone started a review you create a new branch for new translations. It will just avoid people to look at what changed between each commit.

File: reference/var/functions/unserialize.xml [hunk 1]
```diff
@@ -110,6 +110,18 @@
            </simpara>
           </entry>
          </row>
+         <row>
+          <entry><literal>max_depth</literal></entry>
+          <entry><type>int</type></entry>
+          <entry>
+           <simpara>
+            La profondeur maximale des structures autorisée lors de la désérialisation,
```
Comment (adoy) [inline reference/var/functions/unserialize.xml:118]:
```suggestion
            La profondeur maximale autorisée des structures lors de la désérialisation, 
```

File: reference/var/functions/unserialize.xml [hunk 2]
```diff
@@ -151,6 +163,14 @@
       </row>
      </thead>
      <tbody>
+      <row>
+       <entry>7.4.0</entry>
+       <entry>
+        Ajout de l'élément <literal>max_depth</literal> aux
+        <parameter>options</parameter> pour définir la profondeur 
+        maximale des structures autorisées lors de la désérialisation.
```
Comment (adoy) [inline reference/var/functions/unserialize.xml:171]:
```suggestion
        maximale autorisée des structures lors de la désérialisation.
```

File: reference/var/functions/var-export.xml [hunk 3]
```diff
@@ -249,7 +256,7 @@ object(A)#2 (2) {
   </note>
   <warning>
    <para>
-    Lorsque la fonction <function>var_export</function> exporte des objets,
+    Avant PHP 8.2.0, lorsque <function>var_export</function> exporte des objets,
```
Comment (adoy) [inline reference/var/functions/var-export.xml:259]:
```suggestion
    Antérieur à PHP 8.2.0, lorsque <function>var_export</function> exporte des objets, 
```

File: reference/xml/functions/xml-get-error-code.xml [hunk 4]
```diff
@@ -35,9 +35,8 @@
  <refsect1 role="returnvalues">
   &reftitle.returnvalues;
   <para>
-   <function>xml_get_error_code</function> retourne &false;
-   si <parameter>parser</parameter> n'est pas une référence à une analyseur
-   XML valide ou, sinon, retourne un des codes erreur listés dans la section
+   <function>xml_get_error_code</function> retourne
```
Comment (adoy) [inline reference/xml/functions/xml-get-error-code.xml:38]: `retourne` is duplicated

**Comment by victor-prdh** [comment]
> I think it's easier if once someone started a review you create a new branch for new translations. It will just avoid people to look at what changed between each commit.

I will do that now, thanks for your proofreading as always :)

File: reference/url/functions/http-build-query.xml [hunk 5]
```diff
@@ -103,6 +103,28 @@
...
+     <row>
+      <entry>&Version;</entry>
+      <entry>&Description;</entry>
+     </row>
+    </thead>
+    <tbody>
+     <row>
+      <entry>8.0.0</entry>
+      <entry>
+       <parameter>arg_separator</parameter> est maintenant nullable.
```
Comment (Girgias) [inline reference/url/functions/http-build-query.xml:120]:
```suggestion
       <parameter>arg_separator</parameter> est désormais nullable.
```

Is what is usually used.

File: reference/var/functions/unserialize.xml [hunk 6]
```diff
@@ -110,6 +110,18 @@
            </simpara>
           </entry>
          </row>
+         <row>
+          <entry><literal>max_depth</literal></entry>
+          <entry><type>int</type></entry>
+          <entry>
+           <simpara>
+            La profondeur maximale autorisée des structures lors de la désérialisation,
```
Comment (Girgias) [inline reference/var/functions/unserialize.xml:118]:
```suggestion
            La profondeur maximale autorisée des structures lors de la délinéarisation, 
```

Is technically what is mostly used across the documentation, something we may want to change

**Review by Girgias — COMMENTED** [review]
Thank you!

File: reference/xml/functions/xml-get-current-line-number.xml [hunk 7]
```diff
@@ -36,9 +36,7 @@
   &reftitle.returnvalues;
   <para>
```
Comment (devnexen) [inline reference/xml/functions/xml-get-current-line-number.xml:37]: Should the `<function>...</function>` part be dropped here, as it is supposed to be implicit at this point ?
Nit: `retourne` keyword is present twice.

File: reference/var/functions/unserialize.xml (hunk 6 above)
Comment (victor-prdh) [reply reference/var/functions/unserialize.xml:118]: ### edit
I just wrote all my answer and when i read again your change's message i'm not sure i understand it well. 
So for the following message, i just assume you say "we use désérialisation more but thinking to change it in favor of délinéarisation" ! If its the opposite, i juste agree with you.
____

Oh really ? Imo "désérialisation" and "sérialisation" are good. Most of french devs/docs use these terms. Even wikipédia have a real page in french for `sérialisation` in dev but a tiny one for "linéarisation" (and not fully for the dev). Last of all, `sérialisation` is more like real function name of PHP and may be more convenient for new devs to look about `sérialisation` process.

After some check, the only con (and it's a big one) i see is that french doc about `serialise` function already use "linéarisation".

I will try to remember using "linéarisation" next time but i keep some doubt about this !


_______________________

See:
[sérialisation wikipédia](https://fr.wikipedia.org/wiki/S%C3%A9rialisation)
[linéarisation wikipédia](https://fr.wikipedia.org/wiki/Lin%C3%A9arisation)
[sérialisation JS / MDN](https://developer.mozilla.org/fr/docs/Glossary/Serialization)
[sérialisation C# / microsoft](https://learn.microsoft.com/fr-fr/dotnet/csharp/programming-guide/concepts/serialization/)

File: reference/xml/functions/xml-get-current-line-number.xml (hunk 7 above)
Comment (victor-prdh) [reply reference/xml/functions/xml-get-current-line-number.xml:37]: No you're right, just checked and not even in EN doc ! Its wasn't highlighted by the revcheck tool so i just missed it, my bad

File: reference/var/functions/unserialize.xml (hunk 6 above)
Comment (Girgias) [reply reference/var/functions/unserialize.xml:118]: Like I said, this is something we may want to change in a big separate PR (and also update TRANSLATION.txt).

**Review by Girgias — APPROVED** [review]

**Pull request merged**